### DISCUSSION TITLE
Load Video

### DISCUSSION THREAD

**Discussion by MilanNedoma** [bonsai-rx/bonsai#1615] [Q&A]
Hi everyone!

I am just trying to work on a looming stimulus inside Bonsai and I just wanted to ask, whether there is a way to simply play a video when a condition is true. I have set everything up already, and I have even tried to make a custom looming stimulus using the BonVision package, however, I wanted to see whether I could just load and play a video when needed. I have looked around a bit and found the "LoadVideo" node. After testing out multiple settings, I have still not managed to get a video to load and play when needed. 
My question therefore is, am I using the right node? And if so, could someone point me in a direction to get .avi video format to play? If I am very far of, I would also like to ask whether there would be a dedicated package to this issue.

Thank you very much and have a great rest of the day.

**Comment by bruno-f-cruz** [comment]
Can you share a minimal example of what you have working? Thanks!

**Comment by chiyu1203** [comment]
> Hi everyone!
> 
> I am just trying to work on a looming stimulus inside Bonsai and I just wanted to ask, whether there is a way to simply play a video when a condition is true. I have set everything up already, and I have even tried to make a custom looming stimulus using the BonVision package, however, I wanted to see whether I could just load and play a video when needed. I have looked around a bit and found the "LoadVideo" node. After testing out multiple settings, I have still not managed to get a video to load and play when needed. My question therefore is, am I using the right node? And if so, could someone point me in a direction to get .avi video format to play? If I am very far of, I would also like to ask whether there would be a dedicated package to this issue.
> 
> Thank you very much and have a great rest of the day.

In terms of what you want to do with your experiment, you can have a look on the following workflow and there are more [discussion](https://groups.google.com/g/bonsai-users/c/wYd6sYbqw80/m/Z6Yvan9-GAAJ) on the old google group about delivering stimuli based on condition. You can try to find my post there.

Unfortunately, I have never used LoadVideo so I can not help more with that

[sweep_loom_assay_closed_loop.zip](https://github.com/bonsai-rx/bonsai/files/13326548/sweep_loom_assay_closed_loop.zip)

Best Wishes

**Comment by sebus3456789** [comment]
This video explains how you can load and play a video in Bonsai: https://www.youtube.com/watch?v=gYlgBgvSrmg&ab_channel=BonVision
